CHANGELOG.md

```diff
@@ -4,6 +4,72 @@ All notable changes to this project will be documented in this file.
+
+### Changed
+
+- Pull secret documentation now points to
+  [try.openshift.com](https://try.openshift.com) for pull-secret
+  acquisition, instead of pointing at `account.coreos.com`.  Users
+  should update their pull secrets, as old CoreOS pulls secrets are
+  likely to stop working with future installer releases.  For this
+  release, either `try.openshift.com` or `account.coreos.com` pull
+  secrets will work.
+- If the automatic bootstrap teardown (which landed in 0.4.0) times
+  out waiting for the `bootstrap-complete` event, the installer exits
+  with a non-zero exit code.  We had ignored watcher timeouts in 0.4.0
+  due to concerns about watcher robustness, but the current watcher
+  code has been reliable in our continuous integration testing.
+- The etcd service now uses [selectors][kube-selector] to determine
+  the pods it exposes, and the explict etcd endpoints object is gone
+  (replaced by the one Kubernetes maintains based on the selector).
+- On AWS, both masters and worker have moved from t2.medium nodes
+  m4.large nodes (more on AWS instance types
+  [here][aws-instance-types] to address CPU and memory constraints.
+- On AWS, master volume size has been bumped from 30 GiB to 120 GiB to
+  increase our baseline performance from on [gp2's sliding IOPS
+  scale][aws-ebs-gp2-iops] from the 100 IOPS floor up to 360 IOPS.
+  Volume information is not currently supported by [the cluster-API
+  AWS provider's
+  `AWSMachineProviderConfig`][cluster-api-provider-aws-012575c1-AWSMachineProviderConfig],
+  so this change is currently limited to masters created by the
+  installer.
+- On libvirt, we pass ignition information for masters and workers via
+  secrets instead of passing a libvirt volume path.  This makes the
+  libvirt approach consistent with how we already handle AWS and
+  OpenStack.
+- Lots of internal cleanup, especially around trimming dead code.
+
+### Fixed
+
+- The `.openshift_install.log` addition from 0.4.0 removed Terraform
+  output from `--log-level=debug`.  We've fixed that in 0.5.0; now
+  `.openshift_install.log` will always contain the full Terraform
+  output, while standard error returns to containing the Terraform
+  output if and only if `--log-level=debug` or higher.
+- On AWS teardown, errors retrieving tags for S3 buckets and Route 53
+  zones are no longer fatal.  This allows the teardown code to
+  continue it's exponential backoff and try to remove the bucket or
+  zone later.  It avoids some resource leaks we were seeing due to AWS
+  rate limiting on those tag lookups as many simultaneous CI jobs
+  searched for Route 53 zones with their cluster's tags.  We'll still
+  hit those rate limits, but they no longer cause us to give up on
+  reaping resources.
+- On OpenStack, similar retry-during-teardown changes were made for
+  removing ports and for removing subnets from routers.
+- On libvirt, Terraform no longer errors out when launching clusters
```

Comment:
What was this bug?

Reply:
> What was this bug?

More detail in #756.  Or did you want me to include `index 1 out of range for list libvirt_ignition.master.*.id` or something so folks who had hit the bug will recognize this entry more easily?